config.js

```diff
@@ -1,7 +1,8 @@
 (function () {
   var options = {}
-  process.argv.forEach(function (val, idx, arr) {
-    var matches = val.match(/^dotenv_config_(.+)=(.+)/)
+  // skip first argb, which is node command
+  process.argv.slice(1).forEach(function (val) {
```

Comment:
again, not specially optimized for perf, it really doesn't matter at that scale, it's just more readable I think